functions/src/post-document-comment.ts

```diff
@@ -7,7 +7,7 @@ import { validateUserContext } from "./user-context";
 import { createCommentableDocumentIfNecessary } from "./validate-commentable-document";
 
 // update this when deploying updates to this function
-const version = "1.1.4";
+const version = "1.1.5";
```

Comment:
Technically in an earlier PR 
https://github.com/concord-consortium/collaborative-learning/pull/1803/files#diff-29f8eaa9683f87d0677c91b645610001db08a12573c283fa53137e52a81e2849

We never changed the version #